### ISSUE TITLE
Failed to parse Transaction meta (type '_Map<String, dynamic>' is not a subtype of type 'String?')

### ISSUE THREAD

**Issue by Ltei** [merigo-labs/solana-web3#3]
**Describe the bug**
getBlock fails to parse Meta object (can't parse err, TransactionError)

```
package:solana_web3/src/rpc/models/meta.dart 64:43              new Meta.fromJson
package:solana_web3/src/rpc/models/meta.dart 84:28              Meta.tryFromJson
package:solana_web3/src/rpc/models/transaction_data.dart 60:18  new TransactionData.fromJson
package:solana_web3/src/rpc/models/transaction_data.dart 50:14  TransactionData.parse
dart:_internal                                                  ListIterable.toList
package:solana_common/src/extensions/iterable.dart 16:35        IterableSerializable.fromJson
package:solana_common/src/extensions/iterable.dart 24:23        IterableSerializable.tryFromJson
package:solana_web3/src/rpc/models/block.dart 63:40             new Block.fromJson
package:solana_web3/src/rpc/methods/get_block.dart 28:30        GetBlock.decoder
package:solana_jsonrpc/src/jsonrpc/method.dart 92:48            JsonRpcMethod.response
package:solana_jsonrpc/src/jsonrpc/client.dart 90:47            Client.send

type '_Map<String, dynamic>' is not a subtype of type 'String?'
```

**To Reproduce**
```
import 'dart:convert';
import 'dart:io';

import 'package:flutter_test/flutter_test.dart';
import 'package:solana_web3/solana_web3.dart';

void main() {
  test('test', () async {
    final client = Connection(Cluster.mainnet);

    final minSlot = await client.minimumLedgerSlot();
    final maxSlot = (await client.getHighestSnapshotSlot()).full;

    final slots = await client.getBlocks(
      maxSlot - 5,
      endSlot: maxSlot,
      config: GetBlocksConfig(),
    );

    print('GOT ${slots.length} slots');

    for (final slot in slots) {
      // Exception here
      final block = await client.getBlock(
        slot,
        config: GetBlockConfig(maxSupportedTransactionVersion: 10),
      );
      if (block == null) {
        print('BLOCK IS NULL');
        continue;
      }
      final json = jsonEncode(block.toJson());
      print('GOT JSON');
      break;
    }
  });
}
```

**Expected behavior**
It works

**Additional context**
Library version : 0.1.2

**Comment by merigo-labs** [comment]
I'll look at this tomorrow.

**Comment by merigo-labs** [comment]
Fixed in version `solana_web3: ^0.1.3`

**Comment by Ltei** [comment]
Thanks, it works now
